Makefile

```diff
@@ -491,8 +496,14 @@ CODE_GENERATOR_VERSION ?= v0.28.12
 
 KUTTL ?= $(LOCALBIN)/kuttl
 # k8s deps @ 1.28.3
-KUTTL_VERSION ?= v0.16.0
-
+KUTTL_VERSION ?= v0.19.0
```

Comment:
Please do not bump the kuttl version here; kuttl v0.19.0 uses k8s deps at version 1.31.0

Reply:
reverted